lumicks/pylake/kymotracker/kymotrack.py

```diff
@@ -336,6 +336,22 @@ def plot_msd(self, max_lag=None, **kwargs):
     def estimate_diffusion(self, method, max_lag=None):
         r"""Estimate diffusion constant
 
+        There are three algorithms to determine diffusion constants:
+
+            - CVE: Covariance based estimator. Works directly on the tracks.
+            - OLS: Ordinary least-squares estimator based on Mean Squared Displacements (MSD).
+            - GLS: Generalized least-squares estimator based on Mean Squared Displacements (MSD).
+
+        1. Covariance based estimator (CVE)
+
+        The covariance-based diffusion estimator provides a simple unbiased estimator of diffusion.
+        This estimator was introduced in the work of Vestergaard et al [5]_. The correction for
+        missing data was introduced in [6]_. The CVE is unbiased and practically optimal when the
+        signal-to-noise ratio (SNR) is bigger than 1. In this context, the SNR is defined
+        as: :math:`\sqrt{D \Delta t} / \sigma`.
+
+        2. MSD-based estimators
```

Comment:
might be good to specifically state which ones these are (even though it's probably obvious)
```suggestion
        2. MSD-based estimators (OLS, GLS)
```